interfaces/login_ui_endpoints/v0/schema.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2023 Canonical
+# See LICENSE file for licensing details.
+"""This file defines the schemas for the provider and requirer sides of the login_ui_endpoints interface.
+It exposes two interfaces.schema_base.DataBagSchema subclasses called:
+- ProviderSchema
+- RequirerSchema
+Examples:
+    ProviderSchema:
+        unit: <empty>
+        app: {"consent_url": "http://identity-platform-login-ui-url/consent",
+              "error_url": "http://identity-platform-login-ui-url/error",
+              "index_url": "http://identity-platform-login-ui-url/index",
+              "login_url": "http://identity-platform-login-ui-url/login",
+              "oidc_error_url": "http://identity-platform-login-ui-url/oidc_error",
+              "registration_url": "http://identity-platform-login-ui-url/registration",
+              "default_url": "http://identity-platform-login-ui-url"
+              }
+"""
+
+from pydantic import BaseModel
+
+from interfaces.schema_base import DataBagSchema
+
+
+class MyProviderAppData(BaseModel):
+    consent_url: str  # Endpoint Hydra forwards users for consent related operations.
```

Comment:
we haven't really gotten this far yet, but eventually these schemas might be visible from somewhere (say, charmhub). So all the metadata we can get contributes to future-proofing this code.
```suggestion
    consent_url: str = pydantic.Field(
      description="Endpoint Hydra forwards users for consent related operations."
    )
```

Not necessarily something you need to do now, but it might mean less work for later :)

Reply:
@simskij wdyt?

Reply:
Looks good. Will do the fields like this.